images/pom.xml

```diff
@@ -159,6 +182,40 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>eap</id>
+            <properties>
+                <!-- Image to use -->
+                <image.name.wildfly.runtime>registry.redhat.io/jboss-eap-8/eap8-openjdk17-runtime-openshift-rhel8:latest</image.name.wildfly.runtime>
+                <!-- Plugin coordinates -->
+                <server.maven.plugin.groupId>org.jboss.eap.plugins</server.maven.plugin.groupId>
+                <server.maven.plugin.artifactId>eap-maven-plugin</server.maven.plugin.artifactId>
+                <server.maven.plugin.version>${version.org.jboss.eap.plugins.eap-maven-plugin}</server.maven.plugin.version>
+                <!-- Other -->
+                <database.feature.pack.gav>org.jboss.eap:eap-datasources-galleon-pack:8.0.0.Final-redhat-00014</database.feature.pack.gav>
+                <!-- EAP Base properties -->
+                <server.feature.pack.gav>org.jboss.eap:wildfly-ee-galleon-pack:8.0.0.GA-redhat-00011</server.feature.pack.gav>
+                <cloud.feature.pack.gav>org.jboss.eap.cloud:eap-cloud-galleon-pack:1.0.0.Final-redhat-00008</cloud.feature.pack.gav>
+                <cloud.feature.pack.default.config.layer>cloud-default-config</cloud.feature.pack.default.config.layer>
```

Comment:
Removed those lines from `eap` and `xp` profiles. All profiles use `cloud-server` now. I kept `cloud.feature.pack.default.config.layer` in case we will want to overide it, which can happen in theory.